src/components/Contacts/ContactsTable/ContactsTable.test.tsx

```diff
@@ -127,11 +161,19 @@ describe('ContactFilters', () => {
   });
 
   it('simulate row click with searchTerm', async () => {
```

Comment:
@OzzieOrca Not sure what's up with this test. It says the ```onContactSelected``` function is not being called. But the one above works fine. If I remove everything related to ```searchTerm``` in this test, then it works. Not sure what the connection is and why the search term would affect it. It appears to works fine actually using it in the browser.

Reply:
@TheNoodleMoose outside of the test failing everything looks good to me.

Reply:
Idk what's going on with the test. It never fires the click event on the contact row. It almost seems like after the click the whole page rerenders as the row isn't there right after the click but is again after the next waitFor...